### ISSUE TITLE
Optional @property is generated without []

### ISSUE THREAD

**Issue by ozum** [jsdoc2md/jsdoc-to-markdown#145]
## Description

Optional properties (`@property`) are generated without `[]`. (Optional parameters (`@param`) are generated with `[]`)

According to [documentation of `@property` tag](http://usejsdoc.org/tags-property.html), optional properties are valid.

## Input

```js
/**
 * Token Interface.
 * @interface Person
 * @property {string} name - Name of person
 * @property {number} [age] - Age of person (Optional)
 */
```

## Expected
(HTML tags are removed for simplicity)

| Param | Type | Description |
| --- | --- | --- |
| name | string | Name of person |
| [age] | number | Age of person (Optional) |

## Got
(HTML tags are removed for simplicity)

| Param | Type | Description |
| --- | --- | --- |
| name | string | Name of person |
| age | number | Age of person (Optional) |

**Comment by 75lb** [comment]
verified, thanks

**Comment by 75lb** [comment]
fixed in dmd v3.0.10, reinstall jsdoc2md to pick up the change 👍
